### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by geoffroyleconte**

### CODE REVIEW

File: src/qpmodel.jl [hunk 1]
```diff
@@ -13,6 +13,22 @@ end
 
 abstract type AbstractQuadraticModel <: AbstractNLPModel end
 
+"""
+    qp = QuadraticModel(c, Hrows, Hcols, Hvals; Arows = Arows, Acols = Acols, Avals = Avals, 
+                        lcon = lcon, ucon = ucon, lvar = lvar, uvar = uvar)
+
+    qp = QuadraticModel(c, H; A = A, lcon = lcon, ucon = ucon, lvar = lvar, uvar = uvar)
+
+Creates a Quadratic model ``min ~\\tfrac{1}{2} x^T Q x + c^T x + c_0`` with optional bounds
```
Comment (dpo) [inline src/qpmodel.jl:22]:
```suggestion
Create a Quadratic model ``min ~\\tfrac{1}{2} x^T Q x + c^T x + c_0`` with optional bounds
```

File: src/qpmodel.jl [hunk 2]
```diff
@@ -13,6 +13,22 @@ end
...
+`lvar ≦ x ≦ uvar` and optional linear constraints `lcon ≦ Ax ≦ ucon`.
+
+You can also use [`QPSReader.jl`](https://github.com/JuliaSmoothOptimizers/QPSReader.jl) to
+create a Quadratic model:
```
Comment (dpo) [inline src/qpmodel.jl:26]:
```suggestion
create a Quadratic model from a QPS file:
```

**Review by dpo — CHANGES_REQUESTED** [review]

**Review by dpo — APPROVED** [review]

**Comment by dpo** [comment]
Thanks!

**Comment by geoffroyleconte** [comment]
@dpo can you generate the key? I don't think the link is working

**Comment by dpo** [comment]
It's working but the address in the badge is incorrect: https://juliasmoothoptimizers.github.io/QuadraticModels.jl/dev/

**Pull request merged**